exporter/datadogexporter/factory.go

```diff
@@ -236,6 +237,7 @@ func (f *factory) createDefaultConfig() component.Config {
 		HostMetadata: HostMetadataConfig{
 			Enabled:        true,
 			HostnameSource: HostnameSourceConfigOrSystem,
+			sourceTimeout:  31 * time.Second,
```

Comment:
Where does 31 seconds come from? I don't think we need to set a default value for improving tests timeout. 

It may be good for certain users, but I would prefer to not do this until we have a reason to do it, since it may break people

Reply:
1+ the tcp timeout lol.  my primary goal was to not change the behavior of the code unless we really wanted to.. but I realize that this is unnecessary because I already branch on whether to timeout or not